go.mod

```diff
@@ -1,57 +1,57 @@
 module github.com/tektoncd/triggers
 
-go 1.19
+go 1.21
```

Comment:
should we add/update a doc with golang min version requirement and k8s min version requirement ?

Reply:
We don't mention Golang min version. We just give build image which can run. Do we need k8s min doc? We are dependent upon Pipelines LTS and follow whatever version it follows. Probably, we can add a new section later on.

Reply:
Yes I meant section for k8s version